docs/data-sources/ccm_private_certificate_export.md

```diff
@@ -61,4 +65,9 @@ In addition to all arguments above, the following attributes are exported:
   
 * `signed_and_enveloped_data` - Indicates the National Security GMT0010 Standard Specification for
   Encrypting Private Keys - Signature Digital Envelope.
-  
\ No newline at end of file
+
+* `keystore_pass` - Indicates the keystore password. If argument "password" passed in, it will be empty.
+
+* `server_pfx` - Indicates the certificate file with certificate chain file. Encoding by base64.
```

Comment:
the description of this parameter is not correct.

Reply:
down